### DISCUSSION TITLE
Permission stuck on old user

### DISCUSSION THREAD

**Discussion by octvs** [termux/termux-app#3593] [Q&A]
Hey,

I reinstalled Termux for a reason. And now the files under `~/storage/shared/` are all owned by the old user name. 

Things I tried:
- Executing `termux-setup-storage`
- Revoking storage permission and granting
- `chown -R` 

Anybody has an idea?

**Comment by sylirre** [comment]
Use `chown -Rh` instead. By default ownership and group is not set to symbolic links but rather to target.

**Reply by octvs** [reply]
Not working. The issue probably is due to write permissions on the shared storage.
